Update instructions about our logo (#173)
stan-dev/stan-dev.github.io: about/logo/index.md

  creditlink:
---

The Stan name and logo are registered trademarks of [NumFOCUS](http://numfocus.org/)
under the direction of the [Stan Governing Body](/about#stan-governing-body-sgb).

> Both may be freely used when referencing Stan, such as in blog posts, lecture notes,

Reviewer: Replace "Both" with "The Stan name and Stan logo" (repeating "Stan").  

"such as" -> "for example, "

Author: ```suggestion
Both may be freely used when referencing Stan, for example, in blog posts, lecture notes,
```

> or opensourced software packages, but they may not be used in the branding of commercial entities,

Reviewer: "opensourced" -> "open source"

Author: ```suggestion
or open source software packages, but they may not be used in the branding of commercial entities,
```

> such as paid software packages or paid courses, without the express license of the Stan leadership board.

Reviewer: "leadership board" -> "Governing Body"

Author: ```suggestion
such as paid software packages or paid courses, without the express license of the Stan Governing Body.
```

If you have any questions or would like to inquire about licensing the Stan name

> and/or logo for a commercial venture then please contact NumFOCUS.

Reviewer: This needs to say how to contact NumFOCUS.

Author: ```suggestion
and/or logo for a commercial venture then please contact NumFOCUS at <mailto:admin@numfocus.org>.
```

> Under in no circumstances our name and logo cannot be used in ways that appear (to a casual observer) official, affiliated, or endorsed by Stan, unless you have written permission from the Stan Governing Body.

Reviewer: This needs to be written---it's not grammatical English and uses a confusing double negation.  The "no circumstances" is redundant.  Also, it's "may" when talking about permission vs. "can" when talking about ability.  Stan is software, so I think you mean the Stan project here.

So I'd recommend changing this to:

```
The Stan name and Stan logo may not be used in ways that suggest the usage is endorsed by the Stan project without written permission from the Stan Governing Body.
```

Author: ```suggestion
The Stan name and Stan logo may not be used in ways that suggest the usage is endorsed by the Stan project without written permission from the Stan Governing Body.
```
